js/siteorigin-panels/dialog/row.js

```diff
@@ -749,7 +749,7 @@ module.exports = panels.view.dialog.extend({
 	 */
 	deleteHandler: function () {
 		// Trigger a destroy on the model that will happen with a visual indication to the user
-		this.model.trigger('visual_destroy');
+		this.visualDestroyModel();
```

Comment:
I don't see a `visualDestroyModel` function anywhere in this view? I'm guessing it's meant to call the `visualDestroyModel` function in the row view. Regardless this should just be calling `this.model.destroy()` and any other actions that need to happen should listen for the `destroy` event on the model.

Reply:
You're right. That should be getting called on the row view, not the dialog view. Will fix.

`visualDestroyModel` destroys the model with a fade out as a visual cue, there are other times when the model is destroyed and we don't want a visual cue. Like when the row is removed and we need to remove all its child widgets. Using a view function was a work around for all this.

If we can find a way to pass an argument to `model.destroy()` that gets passed to the view, then we can avoid `visualDestroyModel`.